content/introduction/what-is-dolt.md

```diff
@@ -0,0 +1,46 @@
+---
+title: What is Dolt
+---
+
+## Version Controlled Database
+Dolt is a version controlled relational database. It has a SQL query interface, which draws heavily on the MySQL dialect. It implements Git-like version control features.
+
+### Offline
+When Dolt is "offline", it looks very much like Git. Let's use `dolt clone` to acquire a database:
+```
+$ dolt clone dolthub/ip-to-country
+cloning https://doltremoteapi.dolthub.com/dolthub/ip-to-country
+32,832 of 32,832 chunks complete. 0 chunks being downloaded currently.
+```
+
+We now have acquired a database, and we can move into the newly created directory to give `dolt` the right database context:
+```
+$ cd ip-to-country
+```
+
+### Online
+In the previous section we used Dolt's version control features, familiar from Git, to clone a remote database. In [Why Dolt](why-dolt.md) we identified the value of a familiar SQL query interface, served over a widely adopted wire protocol, MySQL.
+
+We can now take the cloned Dolt database "online" by firing up the Dolt SQL Server:
+```
+dolt sql-server
+Starting server with Config HP="localhost:3306"|U="root"|P=""|T="28800000"|R="false"|L="info"
+```
+
+We now have a running Dolt SQL Server that presents a superset of the MySQL dialect as its query interface.
```

Comment:
Show a MySQL client connecting to it on the command line. Like:

`/usr/bin/mysqlclient -q` 

or whatever